toml.md

```diff
@@ -835,19 +841,19 @@ type.name = "Nail"
 Array of Tables
 ---------------
 
-The last syntax that has not yet been described allows writing arrays of tables.
-These can be expressed by using a table name in double brackets. Under that, and
-until the next table or EOF are the key/values of that table. Each table with
-the same double bracketed name will be an element in the array of tables. The
-tables are inserted in the order encountered. A double bracketed table without
-any key/value pairs will be considered an empty table.
+The last syntax that has not yet been described allows writing arrays of
+tables. These can be expressed by using a header with a name in double
+brackets. The first instance of that header defines the array and its
+first table element, and each subsequent instance creates and defines a new
+table in that array. The tables are inserted into the array in the order
```

Comment:
Better: "table element in" instead of "table in".

Reply:
You're absolutely right.